azure/packages/azure-local-service/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@fluidframework/azure-local-service",
-  "version": "0.1.0",
```

Comment:
Does this mean we will synchronize release cadence and version numbering between azure-local-service and the other azure packages?  I see the previous pipeline yml is deleted, so it looks like they will release as a group?

If so, I think that is probably fine but just thinking a bit about their underlying dependencies (client packages vs. tinylicious/server packages) rev'ing at different cadences.

Reply:
Yes, it will be synchronized. If we ever go down the line where it warrants to move it somewhere else (back to server or separate repo), we can certainly do that at that time. But for what it stands today (more on that in the description), it seems reasonable to rev it as part of azure group.